test/addons-napi/test_buffer/test_buffer.c

```diff
@@ -12,7 +12,7 @@
 
 #define NAPI_CALL(env, theCall)                                           \
   if (theCall != napi_ok) {                                               \
-    const char *errorMessage = napi_get_last_error_info()->error_message; \
+    const char* errorMessage = napi_get_last_error_info()->error_message; \
```

Comment:
I don't remember fixing linter errors. It'd be nice to do these things systematically though. For JS there's make jslint which at least points out all the errors and causes the build to fail. Not so for C++.